.storybook/preview.js

```diff
@@ -0,0 +1,57 @@
+import React, { Fragment } from 'react';
+import {
+  ThemeProvider,
+  createMuiTheme,
+  useTheme,
+  makeStyles,
+} from '@material-ui/core/styles';
+import Slider from '@material-ui/core/Slider';
+import Select from '@material-ui/core/Select';
+import FormControl from '@material-ui/core/FormControl';
+import Typography from '@material-ui/core/Typography';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import Paper from '@material-ui/core/Paper';
+import { TASKCLUSTER_THEME } from '../src/utils/theme';
+
+export const globalTypes = {
```

Comment:
`globalTypes` is unused as well as most of the imports above.

Reply:
Hah, I guess that file's not linted!

Reply:
oh, `globalTypes` is a storybook thing (it's exported).  I don't really understand it, but it works :)